zombies.js

```diff
@@ -110,7 +134,13 @@
  * @param {Item/Weapon/Food} item   The item to take.
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
-
+Player.prototype.takeItem = function(item) {
+  if(this.getPack().length <3) {
```

Comment:
This doesn't handle the case where a player is allowed to store similar item (items with the same name).